### ISSUE TITLE
training/test data from steve morrell corpus

### ISSUE THREAD

**Issue by sbenthall** [sbenthall/deeptune#9]
For training of neural network models, I need training/test data for music.

Goal:
 - [x] Start with a directory of MP3s*
 - [x] Break each MP3 into fragments -- parameter: length of time
 - [x] MP3 fragment into array of data
 - [x] arrays of data as X, with song/file title as Y, category label.
 - [x] Export to file
 - [ ] Import into tensorflow

* for training, can use genre diverse openly available music, such as the Steve Morrell corpus.
